cadquery/cq.py

```diff
@@ -413,14 +413,13 @@ def toOCC(self) -> Any:
 
         return self.val().wrapped
 
-    @deprecate_kwarg("centerOption", "ProjectedOrigin")
```

Comment:
I'm assuming we want to leave it for future use so it's probably not a problem, but it looks like `deprecate_kwarg` won't be used anywhere after this removal.

Reply:
Correct, it'll be neede sooner or later.